src/key_info_managers/mod.rs

```diff
@@ -363,13 +395,14 @@ pub struct KeyInfoManagerFactory {
 
 impl KeyInfoManagerFactory {
     /// Create a KeyInfoManagerFactory
-    pub fn new(config: &KeyInfoManagerConfig) -> Result<Self> {
+    pub fn new(config: &KeyInfoManagerConfig, default_auth_type: AuthType) -> Result<Self> {
```

Comment:
Maybe an `Option` is well suited here as this field can be clearly ignored for other KIM

Reply:
Yes, probably best. I'm sure I'll run into having to change it when I add the `SQLiteKeyInfoManager` to the `KeyInfoManagerFactory`